app/services/build_runner.rb

```
@@ -28,7 +28,7 @@ def violations
   end
 
   def priority_violations
-    violations.take(ENV["MAX_COMMENTS"].to_i)
+    violations.take(ENV.fetch("MAX_COMMENTS").to_i)
```

@srt32 Do you mean something like `config/constants.rb`:
```
MAX_COMMENTS = ENV.fetch("MAX_COMMENTS").to_i
RESQUE_RETRY = ENV.fetch("RESQUE_RETRY", 30).to_i
```

@Koronen I think either the default, or the approach like above would be good. If we don't default, we want to fail fast so the user knows to set that variable. Otherwise, the failure won't happen until some point later, when Hound tries to comment.